MBBSEmu/Program.cs

```diff
@@ -262,24 +264,17 @@ static void Main(string[] args)
                     return;
                 }
 
+                //Setup and Run Host
+                var host = _serviceResolver.GetService<IMbbsHost>();
+                host.Start(_moduleConfigurations);
```

Comment:
Move API Reporting before calling Start() and you won't have to worry about calling Stop() within the report

Reply:
The problem is that the modules aren't initialized until we get into host now -- so we have to host.Start(_moduleConfiguration) or else the APIREPORT won't be able to run (no modules)

I think APIREPORT depends on knowing where all the files are etc. which requires MBBSModule to be constructed -- unless we want to look at changing it?

Reply:
Moved Stop() to program.cs per your suggestion.